### DISCUSSION TITLE
Is Audiobookshelf integration in the works?

### DISCUSSION THREAD

**Discussion by Joe31388** [music-assistant/support#3187] [Q&A] [answered]
Is Audiobookshelf integration in the works?  I see a check next to it in the feature request poll but unsure what that means. It would be a awesome integration.

**Comment by OzGav** [comment, marked as answer]
We don’t advise any timelines to avoid disappointment. Having said that it is being discussed internally what needs to be added to core to allow for it.

**Comment by patienttruth** [comment]
> Is Audiobookshelf integration in the works?  I see a check next to it in the feature request poll but unsure what that means. It would be a awesome integration. 

I'm looking forward to a formal integration too. Came here looking for an update.

Maybe a temporary solution would work for you that I just figured out.

I added my "media" share from my NAS which is what audiobookshelf downloads all its podcast to. Now all my podcasts are available in Music Assistant and I can stream them to my speakers. 🎉 #winning 

I'm pretty stoked.
